airflow/www/templates/airflow/task_instance.html

```diff
@@ -39,6 +39,9 @@ <h4>
         {{ url_for(endpoint, dag_id=dag.dag_id, task_id=task_id, execution_date=execution_date) }}
       {%- endif -%}
     {% endmacro -%}
+    <li><a href="{{ url_for('Airflow.grid', dag_id=dag.dag_id) }}">
```

Comment:
Nice! Could we include the task_id, dag_run_id, map_index in the params to link back to the same task instance?

Reply:
I tried, but I don't know how to get the `dag_run_id` value 🤔 
Could you help me?

Reply:
I did it! I'm not sure if it's the most optimal way, though.